WebRoot/js/zimbraAdmin/globalconfig/controller/ZaGlobalConfigViewController.js

```diff
@@ -377,6 +377,73 @@ function () {
 			mods[ZaGlobalConfig.A_zimbraMtaBlockedExtension] = "";
 		}		
 	}
+
+	//validate ephemeral backend url
+	var backendLDAP = "ldap";
+	var backendSSDB = "ssdb";
+	var defaulValue = "//default";
+
+	if (ZaItem.hasWritePermission(ZaGlobalConfig.A_zimbraEphemeralBackendURL, tmpObj)) {
+		if (tmpObj.attrs[ZaGlobalConfig.A_zimbraEphemeralBackendURL]) {
+			var backendURL = tmpObj.attrs[ZaGlobalConfig.A_zimbraEphemeralBackendURL];
+			var chunks = backendURL.split(":");
+			
+			if (chunks < 2 || chunks > 3) {
+				throw new AjxException(AjxMessageFormat.format(ZaMsg.ERROR_INVALID_BACKEND_URL, [backendURL]), AjxException.INVALID_PARAM, "ZaGlobalConfigViewController.prototype._saveChanges");
+			}
+
+			//check input backend with currently supported backends (ldap, ssdb)
```

Comment:
lets remove this validation. Our SSDB backend implementation was intended as default and reference implementation, but admins should be free to implement their own server extensions for other backends and use them.